src/Applications/LDAS_App/util/inputs/ASCAT_sm_mask/ascat_mask_maker.F90

```diff
@@ -0,0 +1,199 @@
+! This program reads in a NetCDF file containing ASCAT soil moisture masks available from:
+! Lindorfer, R., Wagner, W., Hahn, S., Kim, H., Vreugdenhil, M., Gruber, A., Fischer, M., & Trnka, M. (2023). 
+! Global Scale Maps of Subsurface Scattering Signals Impacting ASCAT Soil Moisture Retrievals (1.0.0) [Data set]. 
+! TU Wien. https://doi.org/10.48436/9a2y9-e5z14
+!
+! It provides the possibility to combine different masks (default case is combination of subsurface and wetland masks)
+! and interpolates onto a regular grid with a (hardwired) 0.1 degree lat/lon spacing and -90/-180 degree lower left 
+! corner used for quick indexing in the ASCAT observation reader QC routine.
+!
+! Author: AM Fox, March, 2024
+
+program ascat_mask_maker
+
+    use netcdf
+
+    implicit none
+
+    integer :: ncid, varid, dimid, ierr, len, N_gpi, dimids(2)
+    integer :: i, j, closest_index
+
+    integer(kind=1), dimension(:,:), allocatable :: mask_out
+    integer(kind=1)                              :: missing_value
+
+    real, dimension(:), allocatable :: asc_lon, asc_lat, cold_mask, wet_mask, veg_mask, subsurface_mask, combined_mask
+    real, dimension(:), allocatable :: lon, lat, distances
+    real                            :: d_lon, d_lat, ll_lon, ll_lat
+
+    character(400)                  :: fname_in
+
+    ! --------------------------------------------------------------------------------
+    !
+    ! hardwired variables
+
+    ! ASCAT soil moisture mask file from Lindorfer et al 2023 
+    
+    fname_in = '/discover/nobackup/amfox/subsurface_scattering_ASCAT_ERA5_Land.nc'    
```

Comment:
@amfox37, @biljanaorescanin : We'll need to find a project disk where we store this type of input file.  It should not live in a user's directory.

Reply:
@amfox37 :  For lack of a better idea, I suggest we add the Lindorfer et al 2023 mask file into the ASCAT EUMETSAT obs dir.   But I don't know where the files are.  The data dir from the `LDASsa_DEFAULT_inputs_ensupd.nml` file does not seem to exist, which would mean that we also need to fix the nml file:
`/discover/nobackup/projects/gmao/merra/iau/merra_land/DATA/ASCAT_EUMETSAT/`
cc: @gmao-qliu @biljanaorescanin

Reply:
The observations are here:
/discover/nobackup/projects/gmao/smap/SMAP_Nature/ASCAT_EUMETSAT

In the past we discussed if this was a suitable home - and concluded it was OK for now. I can update the default namelists if that is still the case.